common-protobuf/src/test/java/io/protostuff/runtime/model/ModelProtobuf.java

```diff
@@ -0,0 +1,2858 @@
+// Generated by the protocol buffer compiler.  DO NOT EDIT!
+// source: ModelProtobuf.proto
+
+package io.protostuff.runtime.model;
```

Comment:
auto generated code shall not be committed, if it can be generated during the build